src/Icon/Icon.tsx

```diff
@@ -1268,6 +1268,20 @@ export const Icon: FC<Props> = ({
         />
       </svg>
     )}
+    {render === 'excluded' && (
+      <svg width="100%" height="100%" viewBox="0 0 32 32" fill="none">
+        <path
+          fillRule="evenodd"
+          clipRule="evenodd"
+          d="M16 32C24.8366 32 32 24.8366 32 16C32 7.16344 24.8366 0 16 0C7.16344 0 0 7.16344 0 16C0 24.8366 7.16344 32 16 32Z"
+          fill={theme.colors.error}
```

Comment:
can we make these use the colour prop instead? better keep them generic even tho they are used with only red and green colour just in case there's gonna be a different use case where we need them with the default secondary background colour. Also the render name should match what designers uses in Figma so can you please ask Luke to add these two in the DS Global Iconography file and name them the same? otherwise it all ends up out of sync

Reply:
Ok cool!